src/components/schedule-page/validation-drawer/error-loader.component.tsx

```diff
@@ -0,0 +1,56 @@
+import { Button } from "../../common-components";
+import React from "react";
+import { ScheduleErrorMessageModel } from "../../../common-models/schedule-error-message.model";
+import ErrorList from "./error-list.component";
+import { errorLoaderState, Props } from "./validation-drawer.component";
+import { SpanErrors } from "./span-errors.component";
+
+interface ErrorLoaderOptions {
+  state?: Props;
+  errors?: ScheduleErrorMessageModel[];
+}
+
+export default function ErrorLoaderComponent(options: ErrorLoaderOptions): JSX.Element {
+  return (
+    <>
+      <div>{options.state?.message}</div>
+      {options.state?.state === errorLoaderState.CHECKING && (
+        <div className="error-loading-container">
+          <div className="center">
+            <div>SPINNER??</div>
+            <div className="error-loading-text">Trwa sprawdzanie planu</div>
+          </div>
+        </div>
+      )}
+      {options.state?.state === errorLoaderState.CONNECTIONFAILED && (
+        <div className="error-loading-container">
+          <div className="center">
+            <img
+              alt=""
+              src="https://firebasestorage.googleapis.com/v0/b/tara-cloud.appspot.com/o/orgs%2Fnurse-scheduling%2Ftasks%2F169%2Fwarning-sign-487a6f66-ab0b-419c-801d-a4d690d1b0da.svg?alt=media&token=ab809143-93d0-4313-9fbf-85778efc23e4"
```

Comment:
Myślę, że będzie lepiej zamiast podawać url, będzie pobrać go i tu zalączyć, ponieważ może być taka sytuacja, że ktoś będzie używać naszej aplikacji nie mając połączenia. 

https://stackoverflow.com/questions/39999367/how-do-i-reference-a-local-image-in-react